### PR TITLE
new pr

### PR DESCRIPTION
**Pull request by SebastianO1717**

### CODE REVIEW

File: Extensions/Program.cs [hunk 1]
```diff
@@ -1,2 +1,46 @@
...
+/// Why extension methods? Extension methods enable you to "add" methods to existing types without creating a new derived type
+/// The extension methods are static methods, but they're called as if they were instance methods on the extended type. 
+/// 
+/// When using an extension method to extend a type whose source code you aren't in control of, you run the risk that a change in the implementation of the type will cause your extension method to break.
+/// 
+/// 
+/// </summary>
+/// 
+// New Comment
+public interface FooInterface
```
Comment (SebastianO1717) [inline Extensions/Program.cs:15]: Nice interface name

File: Extensions/Program.cs [hunk 2]
```diff
@@ -1,2 +1,50 @@
...
+﻿/// <summary>
+/// Why extension methods? Extension methods enable you to "add" methods to existing types without creating a new derived type
+/// The extension methods are static methods, but they're called as if they were instance methods on the extended type. 
+/// 
+/// When using an extension method to extend a type whose source code you aren't in control of, you run the risk that a change in the implementation of the type will cause your extension method to break.
+/// 
+/// 
+/// </summary>asdfasdfsdafasdf
+/// 
+// New Comment my new change
```
Comment (SebastianO1717) [inline Extensions/Program.cs:10]: Please remove all of these comments

File: Extensions/Program.cs [hunk 3]
```diff
@@ -1,2 +1,50 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿/// <summary>
+/// Why extension methods? Extension methods enable you to "add" methods to existing types without creating a new derived type
+/// The extension methods are static methods, but they're called as if they were instance methods on the extended type. 
+/// 
+/// When using an extension method to extend a type whose source code you aren't in control of, you run the risk that a change in the implementation of the type will cause your extension method to break.
+/// 
+///
```
Comment (Sebastian1712) [inline Extensions/Program.cs:7]: remove whitespace

Comment (SebastianO1717) [reply Extensions/Program.cs:7]: I'll go ahead and remove the whitespace

Comment (SebastianO1717) [reply Extensions/Program.cs:7]: I will go ahead and remove the whitespace

Comment (SebastianO1717) [reply Extensions/Program.cs:7]: New comment

File: Extensions/FileTwo.cs [hunk 4]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Extensions
+{
+    internal class FileTwo
```
Comment (Sebastian1712) [inline Extensions/FileTwo.cs:9]: I don't like this file name

Comment (SebastianO1717) [reply Extensions/FileTwo.cs:9]: Why?

File: Extensions/Program.cs [hunk 5]
```diff
@@ -1,2 +1,46 @@
...
+    public string Name { get; set; }
+
+    public FooA()
+    {
+        Name = "FooA";
+    }
+
+    public string GetName()
+    {
+        return Name;
```
Comment (Sebastian1712) [inline Extensions/Program.cs:28]: is this okay?

Comment (SebastianO1717) [reply Extensions/Program.cs:28]: I think so!

**Pull request closed without merging**